apps/web/src/components/GalleryEditor/PiecesSidebar/AddWalletSidebar.tsx

```diff
@@ -68,7 +70,7 @@ export function AddWalletSidebar({ handleRefresh, selectedChain, queryRef }: Pro
             variant="secondary"
             onClick={handleManageWalletsClick}
           >
-            {ctaButtonText}
+            {selectedChain !== 'All Networks' ? ctaButtonText : 'Connect any account'}
```

Comment:
nit: for readability, let's make the truthy case first in the ternary

`{selectedChain === 'All Networks' ? 'Connected any account' : ctaButtonText}`